packages/components/src/components/data-list/data-list-row.tsx

```diff
@@ -1,8 +1,24 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 import { useIsMounted } from '@deriv/shared';
+import { TRowRenderer } from './data-list';
+
+type TDataListRow = {
+    action_desc?: {
+        component: React.ReactNode;
+    };
+    destination_link?: string;
+    row_gap?: number;
+    row_key: string | number;
+    rowRenderer: TRowRenderer;
+    measure?: () => void;
+    is_dynamic_height?: boolean;
+    is_new_row: boolean;
+    is_scrolling: boolean;
+    passthrough?: any;
+    row: any;
```

Comment:
@ali-hosseini-deriv please check the type the row has. it's passed from DataList as data_source[index]🙂

Reply:
@maryia-deriv this `data_source` prop is a dynamic array and it depends on another prop from data-list component (Key mapper), basically it maps the data using keymapper function which is passed to data-list component. so there's no fixed schema for it.

Reply:
done, it's now generic